README.md

```diff
@@ -1 +1,69 @@
-# BlockNLPAlgorithms.jl
\ No newline at end of file
+# BlockNLPAlgorithms.jl
+This package implements several popular decomposition algorithms for block-structured nonlinear optimization problems (NLPs). This package requires the NLPs to be implemented as `AbstractBlockNLPModels`; for more details on modeling block-structured NLPs, please refer to the documentation of [BlockNLPModels.jl](https://github.com/exanauts/BlockNLPModels.jl).
+
+## Installation
+This package is currently under active development and has not been registered yet. However, to access the current code, one can enter the following command in Julia REPL:
+
+```julia
+]add "https://github.com/exanauts/BlockNLPAlgorithms.jl"
+```
+
+To confirm if the package was installed correctly, please run:
+```julia
+test BlockNLPAlgorithms
+```
+The test code generates and solves a small instance of a `BlockNLPModel` with all the implemented algorithms using [MadNLP.jl](https://github.com/MadNLP/MadNLP.jl) as the subproblem solver.
```

Comment:
Suggestion:
`with all the implemented algorithms` -> `with both dual decomposition and ADMM, using [MadNLP](...) to solve the suproblems.`